src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -3,416 +3,11 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 package com.mozilla.telemetry
 
-import java.sql.Timestamp
-
 import com.mozilla.telemetry.heka.Message
-import com.mozilla.telemetry.pings.Meta._
-import org.joda.time.Months
-import org.joda.time.format.DateTimeFormat
-import org.json4s.JsonDSL._
 import org.json4s._
 import org.json4s.jackson.JsonMethods._
 
-import scala.util.{Success, Try}
-
 package object pings {
```

Comment:
We could consider renaming this package object now.

Reply:
How about getting rid of the package object?
Since these conversion methods are used only in concrete ping classes, I moved them to `Ping` companion object.